zad4/app/services/football_player_manager.rb

```diff
@@ -8,14 +8,15 @@ def initialize(player)
   end
 
   def stronger_players
-    # TODO: Implement me pls
+    Player.where("power > #{player.power}")
```

Comment:
W takich `where` przyjęło się korzystanie z labeli i zanków zapytania do interpolowania wartości do zapytań sql :) Tutaj jest okej, ale łatwo się machnąć i pzowolić w ten sposób na sql injection, taka interpolacja nie uchodzi za bezpieczną w kwestii zapytań :)

Czyli powinno to wyglądać tak:
```ruby
Player.where('power > ?', player.power)
```

Lub tak:
```ruby
Player.where('power > :power', power: player.power)
```

Reply:
Dzięki za uwagę :) Bezpieczeństwo ważna sprawa :)